runtimes/core/src/sqldb/val.rs

```diff
@@ -253,6 +261,9 @@ impl ToSql for PValue {
     }
 
     fn accepts(ty: &Type) -> bool {
+        if is_pgvector(ty) {
```

Comment:
Maybe move this to the bottom? Seems unnecessary to check for this before the others that are more common